orsopy/fileio/data_source.py

```diff
@@ -82,24 +119,35 @@ class InstrumentSettings(Header):
 
 @dataclass(repr=False)
 class Measurement(Header):
-    """The measurement elements for the header."""
+    """
+    The measurement elements for the header.
+
+    :param instrument_settings: Instrumentation details.
+    :param data_files: Raw data files produced in the measurement.
+    :param references: Raw reference files used in the reduction.
+    :param scheme: Measurement scheme (one of :code:`'angle-dispersive'`,
+        :code:`'energy-dispersive'`/:code:`'angle- and energy-dispersive'`).
+    """
     instrument_settings: InstrumentSettings
     data_files: List[Union[File, str]]
     references: Optional[List[Union[File, str]]] = None
-    scheme: Optional[Union[
-        Literal[
-            "angle- and energy-dispersive",
-            "angle-dispersive",
-            "energy-dispersive",
-        ]
-    ]] = None
+    scheme: Optional[str] = None
```

Comment:
See previous comments on typing (why have the Literals been removed?)

Reply:
See above